src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator start work");
+    }
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stop work");

```

Comment:
Similarly, the message 'Excavator stop work' should be more informative. You could change it to 'Excavator has stopped working' to clearly communicate the state of the machine.